js/openstreet_maps.js

```diff
@@ -94,10 +112,10 @@
       Drupal.islandora_simple_map.markers.push(marker);
     },
     redraw: function(map, bounds) {
-      map.fitBounds(bounds);
-      map.invalidateSize();
-      map.panTo(bounds.getCenter());
-      map.flyToBounds(bounds);
+      //map.fitBounds(bounds);
+      //map.invalidateSize();
+      //map.panTo(bounds.getCenter());
+      //map.flyToBounds(bounds);
```

Comment:
Ummm 🙄 should this all be commented out?

Reply:
Yikes, sorry, will uncomment!